docs/api/rest_api/rest_api_reference/input/ez-commerce.raml

```diff
@@ -1,8 +1,8 @@
-displayName: eCommerce API
+displayName: eCommerce API  (deprecated as of Ibexa DXP 4.4)
```

Comment:
I would discourage this practice as in the left navigation menu it will be rendered as "eCommerce API (deprecated as of Ibexa DXP 44)". It seems to be due to a JS collecting the entries to generate the left navigation.

In #1430, I propose to move the "as of" part to the description to shorten the entry in the left navigation and avoid the missing-dot bug.